pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scanapi"
-version = "2.1.0-rc.1"
+version = "2.1.0-rc.2"
```

Comment:
We've just merge a PR that automates this process, so there is no more need to update the version manually 🎉 . You can leave as "2.1.0" - the same way it is on master.